jupyter_server_fileid/manager.py

```diff
@@ -154,16 +163,15 @@ def move(self, old_path: str, new_path: str) -> None:
         if id:
             self.con.execute("UPDATE Files SET path = ? WHERE path = ?", (new_path, old_path))
         else:
-            cursor = self.con.execute("INSERT INTO Files (path) VALUES (?)", (new_path,))
-            id = cursor.lastrowid
+            id = self._create(new_path)
 
         self.con.commit()
         return id
 
-    def copy(self, from_path: str, to_path: str) -> Optional[int]:
-        cursor = self.con.execute("INSERT INTO Files (path) VALUES (?)", (to_path,))
+    def copy(self, from_path: str, to_path: str) -> Optional[str]:
```

Comment:
Although the methods called during event handling are not public, any FileIdManager implementations that _choose_ to leverage content events should be receiving the entire model that is the event's payload, not just path information.  Since it looks like `ArbitraryFileIdManager` is such an implementation, I think these should take the model (`dict`).

Reply:
`copy()`, `move()`, and `delete()` are intended to be public methods, as that allows developers to perform in-band edits without always having to go through the contents manager. I don't know of any users at the moment, but we want to keep this door open for now.